cmd/kubeadm/app/cmd/phases/bootstraptoken.go

```diff
@@ -17,317 +17,112 @@ limitations under the License.
-
-			// Create the RBAC rules that expose the cluster-info ConfigMap properly
-			err = clusterinfo.CreateClusterInfoRBACRules(client)
-			kubeadmutil.CheckErr(err)
-		},
+	cfg := data.Cfg()
+	skipTokenPrint := data.SkipTokenPrint()
+	kubeconfigPath := data.KubeConfigPath()
+	client, err := data.Client()
+	if err != nil {
+		return err
 	}
-	return cmd
-}
 
-// NewSubCmdNodeBootstrapToken returns the Cobra command for running the node sub-phase
-func NewSubCmdNodeBootstrapToken(kubeConfigFile *string) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:     "node",
-		Short:   "Configures the node bootstrap process",
-		Aliases: []string{"clusterinfo"},
-		Long:    cmdutil.MacroCommandLongDescription,
+	tokens := []string{}
+	for _, bt := range cfg.BootstrapTokens {
+		tokens = append(tokens, bt.Token.String())
 	}
-
-	cmd.AddCommand(NewSubCmdNodeBootstrapTokenPostCSRs(kubeConfigFile))
-	cmd.AddCommand(NewSubCmdNodeBootstrapTokenAutoApprove(kubeConfigFile))
-
-	return cmd
-}
-
-// NewSubCmdNodeBootstrapTokenPostCSRs returns the Cobra command for running the allow-post-csrs sub-phase
-func NewSubCmdNodeBootstrapTokenPostCSRs(kubeConfigFile *string) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "allow-post-csrs",
-		Short: "Configures RBAC to allow node bootstrap tokens to post CSRs in order for nodes to get long term certificate credentials",
-		Long:  nodePostCSRsLongDesc,
-		Run: func(cmd *cobra.Command, args []string) {
-			client, err := kubeconfigutil.ClientSetFromFile(*kubeConfigFile)
-			kubeadmutil.CheckErr(err)
-
-			// Create RBAC rules that makes the bootstrap tokens able to post CSRs
-			err = node.AllowBootstrapTokensToPostCSRs(client)
-			kubeadmutil.CheckErr(err)
-		},
+	if !skipTokenPrint {
+		if len(tokens) == 1 {
+			glog.V(1).Infof("[bootstraptoken] using token: %s\n", tokens[0])
+		} else if len(tokens) > 1 {
+			glog.V(1).Infof("[bootstraptoken] using tokens: %v\n", tokens)
+		}
 	}
-	return cmd
-}
-
-// NewSubCmdNodeBootstrapTokenAutoApprove returns the Cobra command for running the allow-auto-approve sub-phase
-func NewSubCmdNodeBootstrapTokenAutoApprove(kubeConfigFile *string) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "allow-auto-approve",
-		Short: "Configures RBAC rules to allow the csrapprover controller automatically approve CSRs from a node bootstrap token",
-		Long:  nodeAutoApproveLongDesc,
-		Run: func(cmd *cobra.Command, args []string) {
-			client, err := kubeconfigutil.ClientSetFromFile(*kubeConfigFile)
-			kubeadmutil.CheckErr(err)
-
-			// Create RBAC rules that makes the bootstrap tokens able to get their CSRs approved automatically
-			err = node.AutoApproveNodeBootstrapTokens(client)
-			kubeadmutil.CheckErr(err)
 
-			// Create/update RBAC rules that makes the nodes to rotate certificates and get their CSRs approved automatically
-			err = node.AutoApproveNodeCertificateRotation(client)
-			kubeadmutil.CheckErr(err)
-		},
+	// Create the default node bootstrap token
```

Comment:
I think that this phase should give a user a message, otherwise this phase is almost silent about implemented actions. e.g.
```
fmt.Println("[bootstrap-token] Configuring bootstrap tokens,  cluster-info ConfigMap, RBAC Roles")
```